frontend/src/locales/en/translation.json

```diff
@@ -1049,7 +1049,7 @@
       "widgetType": "Widget Type",
       "type": {
         "chart": "Chart",
-        "widgetChart": "Private Chart",
+        "widgetChart": "privateChart",
```

Comment:
why

Reply:
Previously https://github.com/running-elephant/datart/pull/1296 Cuiyansong found issue with this. I was told this was fixed by Datart team but I do not have details.

If I continue with "Private Chart", running npm run test:ci gives error. Hence I reverted to original.

Reply:
ok, let me update and submit PR again.